tests/functional/json/test_make_references.json

```diff
@@ -0,0 +1,14 @@
+{
+    "make_references.mappability_kmer_length": 12,
+    "make_references.reference_fasta": "tests/data/ref/chr22.fa",
+    "make_references.machine_sizes": {
+        "bwa": "small",
+        "fai": "small",
+        "bowtie": "small",
+        "mappable": "small",
+        "chr_buckets": "small",
```

Comment:
Should change all chr -> chrom to be consistent with e.g. https://github.com/ENCODE-DCC/dnase-seq-pipeline/blob/3fce1ab3fc8fab7abc49e578191bd78ee6540e8e/wdl/structs/hotspot2.wdl#L5

Reply:
Actually maybe should be named after subworkflows they are calling. `build_fasta_index`, `get_chrom_info`, etc.

Reply:
yeah, i like naming according to the subworkflow.